apps/institution-service/package.json

```diff
@@ -5,19 +5,39 @@
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
-    "dev": "nodemon ./src/app.ts"
+    "db:setup": "npm run db:create && npm run db:migrate && npm run db:seed",
+    "db:create": "npx sequelize-cli db:create",
+    "db:migrate": "npx sequelize-cli db:migrate",
+    "db:seed": "npx sequelize-cli db:seed:all",
+    "db:reset": "npx sequelize-cli db:drop && npm run db:setup",
+    "test:db:setup": "npm run test:db:create && npm run test:db:migrate && npm run test:db:seed",
+    "test:db:create": "npx sequelize-cli db:create --env test",
+    "test:db:migrate": "npx sequelize-cli db:migrate --env test",
+    "test:db:seed": "npx sequelize-cli db:seed:all --env test",
+    "test:db:reset": "npx sequelize-cli db:drop && npm run test:db:setup --env test",
+    "dev": "nodemon ./app.ts"
   },
   "dependencies": {
     "dotenv": "^16.4.5",
-    "express": "^4.19.2"
+    "express": "^4.19.2",
+    "pg": "^8.12.0",
+    "pg-hstore": "^2.3.4",
+    "reflect-metadata": "^0.2.2",
+    "sequelize": "^6.37.3",
+    "sequelize-cli": "^6.6.2",
+    "sequelize-typescript": "^2.1.6"
   },
   "devDependencies": {
     "@types/express": "^4.17.21",
+    "@types/jest": "^29.5.12",
     "@types/morgan": "^1.9.9",
-    "@types/node": "^20.12.11",
+    "@types/node": "^20.14.12",
+    "@types/validator": "^13.12.0",
     "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
You mentioned in a comment that jsdom wasn't compatible with sequelize. Is this dependency still needed?